app/services/export_activity_to_csv.rb

```diff
@@ -0,0 +1,29 @@
+require "csv"
+
+class ExportActivityToCsv
+  attr_accessor :activity
+
+  def initialize(activity:)
+    @activity = activity
+  end
+
+  def call
+    activity_presenter = ActivityPresenter.new(activity)
+    [
+      activity_presenter.identifier,
+      activity_presenter.sector,
+      activity_presenter.title,
+      activity_presenter.description,
+      activity_presenter.status,
+      activity_presenter.planned_start_date,
+      activity_presenter.actual_start_date,
+      activity_presenter.planned_end_date,
+      activity_presenter.actual_end_date,
+      activity_presenter.recipient_region,
+      activity_presenter.recipient_country,
+      activity_presenter.aid_type,
+      activity_presenter.level,
+      "https://#{ENV["DOMAIN"]}#{Rails.application.routes.url_helpers.organisation_activity_details_path(activity_presenter.organisation, activity_presenter)}",
```

Comment:
Also, might benefit from using the `_url` helper rather than `_path`. Might open the can of `default_url_options`-shaped worms however. But means we could potentially be able to have the links work correctly in development.

Reply:
Good call, I didn't even think of that!